html-forms/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title> CODING FORMS </title>
+</head>
+<body>
+
+  <h1> Coding Form! </h1>
+
+  <form action="">
+
+      <label for="email"> Email </label>
+      <div>
+        <input type="text" id="email">
+      </div>
+
+      <label for="reason"> Why is coding cool? </label>
+      <div>
+        <select name="Select an option" id="reason">
+        <option value="" disabled selected> Select an option </option>
+        <option value="interesting"> Because it's interesting </option>
+        <option value="fun"> Because it's fun </option>
+        <option value="powerful"> Because it's powerful </option>
+      </select>
+      </div>
+
+      <label for="lang"> Favorite Language (choose one) </label>
+      <div>
+        <input type="radio" name="lang" value="HTML"> HTML
```

Comment:
instead of a separate label here, it is better to use a `div` for the text above, and then wrap each of these inputs in a label.